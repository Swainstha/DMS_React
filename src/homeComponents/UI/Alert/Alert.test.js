import testSnapFunction from '../../../util/testSnapFunction';
import Alert from './Alert'
import React from 'react';


testSnapFunction("<Alert/>", "Snapshot test for Alert",<Alert alerts={["alert1","are you sure"]}/> )