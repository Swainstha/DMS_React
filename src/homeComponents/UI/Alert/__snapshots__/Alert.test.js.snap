// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Alert/> Snapshot test for Alert 1`] = `ShallowWrapper {}`;
